### DISCUSSION TITLE
Is Control tab able to show multiple images from the batch in the Gallery view at the moment?

### DISCUSSION THREAD

**Discussion by redrum-llik** [vladmandic/sdnext#2862] [General]
Just decided to give a try to the Diffusers backend and it is rather quick, shaving off 5-8 seconds off my generations compared to original backend on Arc A770 16GB - you guys are great! However, I am not entirely sure about the expected behavior of the Control tab; if I select batch size > 1 or set up multiple batches, pipeline does generate extra images, but only one is shown in the Gallery view:

![image](https://github.com/vladmandic/automatic/assets/63649969/ea388281-532f-4944-a000-90e12ef4f312)

Log output: 

```
Processed: images=4 time=23.60 its=3.39 memory={'ram': {'used': 4.1, 'total': 31.94}, 'gpu':
                         {'used': 5.45, 'total': 15.56}, 'retries': 0, 'oom': 0}
```

At this point of development, is it the expected behavior, or do I have an issue with my setup? For the reference, my extensions are

```
Extensions: enabled=['adetailer', 'Config-Presets',
                         'multidiffusion-upscaler-for-automatic1111', 'sd-dynamic-prompts', 'sd-webui-photopea-embed',
                         'sd-webui-reactor', 'sd-webui-regional-prompter'] extensions
```

Thank you in advance.

**Comment by vladmandic** [comment]
looks like an issue that should be quickly fixable. i've created an issue to track it, see there for updates.

**Comment by vladmandic** [comment]
fixed in latest dev branch.

**Reply by redrum-llik** [reply]
Works great, thank you a lot!
